src/Governance.sol

```diff
@@ -641,12 +642,14 @@ contract Governance is Multicall, UserProxyFactory, ReentrancyGuard, Ownable, IG
         require(_absoluteLQTYVetos.length == _absoluteLQTYVotes.length, "Length");
 
         // To ensure the change is safe, enforce uniqueness
-        _requireNoDuplicates(_initiatives);
         _requireNoDuplicates(_initiativesToReset);
+        _requireNoDuplicates(_initiatives);
 
         // Explicit >= 0 checks for all values since we reset values below
         _requireNoNegatives(_absoluteLQTYVotes);
         _requireNoNegatives(_absoluteLQTYVetos);
+        // If the goal is to remove all votes from an initiative, including in _initiativesToReset is enough
+        _requireNoNOP(_absoluteLQTYVotes, _absoluteLQTYVetos);
```

Comment:
Technically, this renders the previous `_requireNoNegatives()` checks redundant, although we might want to revise the name and revert message of `_requireNoNOP` if we eliminate them.

Reply:
Why? `_requireNoNOP` doesn’t prevent a negative value, only that for each pair at least one is positive.
Anyway, I was thinking we would remove `_requireNoNegatives` when we change the signature `int88` to unsigned. It’s weird to call them absolute but allow them to be negative to then check they are not.

Reply:
Ah yes, I forgot it's `||`.

Agree the interface should be unsigned.